src/imputation/imputation_main.py

```diff
@@ -141,6 +150,7 @@ def run_imputation(
         # create trimming qa dataframe with required columns from schema
         schema_path = config["schema_paths"]["manual_trimming_schema"]
         schema_dict = load_schema(schema_path)
+        schema_dict = add_additional_columns_to_schema(schema_dict, config)
         trimming_qa_output = create_output_df(qa_df, schema_dict)
```

Comment:
are these extra columns actually needed for the trimming_qa_output?

Reply:
but the schema_dict is only for the manual_trimming_schema, so it's not needed.

Reply:
I suggest the only time we might need to see those extra 3 cols is in the mapping QA output, we don't really need them after we create the imputation class, as you noted. What do you think?